tests/serializer_valid.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
```

Comment:
This seems to be redundant and the various other tests already assert this notice isn't emitted for valid serializers